src/vellum/workflows/nodes/displayable/tool_calling_node/tests/test_dynamic_subworkflow_deployment_node.py

```diff
@@ -0,0 +1,101 @@
+from unittest.mock import Mock
+from typing import List
+
+from vellum import ChatMessage
+from vellum.client.types.function_call import FunctionCall
+from vellum.client.types.function_call_vellum_value import FunctionCallVellumValue
+from vellum.client.types.string_chat_message_content import StringChatMessageContent
+from vellum.workflows.nodes.displayable.tool_calling_node.utils import DynamicSubworkflowDeploymentNode
+from vellum.workflows.state.base import BaseState, StateMeta
+from vellum.workflows.state.context import WorkflowContext
+
+
+def test_dynamic_subworkflow_deployment_node__with_function_call_output():
```

Comment:
delete these two tests and this file. Instead, update the test in `basic_tool_calling_node_workflow_deployment` to include a WORKFLOW STREAMING event within `mock_workflow_execution`, but nothing else about the test should change. This change should fail on main and pass after my feedback below